src/patternfly/components/Pagination/pagination.scss

```diff
@@ -255,6 +255,19 @@ $pf-c-pagination--breakpoint-map: build-breakpoint-map();
     }
   }
 
+  // hide number input up/down arrow buttons
+  input[type="number"] {
+    // stylelint-disable
+    &::-webkit-inner-spin-button,
+    &::-webkit-outer-spin-button {
+    // stylelint-enable
+      appearance: none;
```

Comment:
This is great! We do this in number input using an older technique. Any interest in updating that as part of this PR, too? If not we can make that a separate issue:

https://github.com/patternfly/patternfly/blob/d57498cdc0946095da6f605e95067df610f52787/src/patternfly/components/NumberInput/number-input.scss#L21-L29